setup.py

```diff
@@ -39,10 +39,6 @@ def run(self):
         # Build virtme-ng-init
         if build_virtme_ng_init:
             check_call(["make", "init"])
-            check_call(
```

Comment:
Can we make this conditional and execute it only when `platform.system() == "Linux"`  instead of removing it?

Reply:
Ok, I understand it's irrelevant in 99.999% of the cases, but what's the benefit of having an unstripped virtme-ng-init? Do you need this change because strip is not available on macOS? I'm just trying to understand the reason why you dropped this command.

Reply:
Oh sorry, I didn't notice you already explained in the commit message. Alright, if it helps for portability let's just get rid of the strip command.

Reply:
On my side, this binary is 3.9M unstripped. Can we not simply drop `-s`?

Without `-s`, I have the exact same file that is produced on my side (556K, same checksum).

Reply:
Sure, I can do that. I wonder if the difference is that mine is compiled for linux-musl rather than linux-gnu.